docs/eventing/samples/container-source/README.md

```diff
@@ -71,18 +71,21 @@ kind: ContainerSource
 metadata:
   name: test-heartbeats
 spec:
-  image: <heartbeats_image_uri>
+  template:
+    spec:
+      containers:
+        - image: <heartbeats_image_uri>
```

Comment:
Seems container `name` is missed here, this yaml template will result in an error.

Reply:
@Fredy-Z Good catch. Fixed.